packages/terra-slide-panel/examples/Index.jsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropsTable from '../../../packages/terra-site/src/PropsTable';
+import Markdown from '../../../packages/terra-site/src/Markdown';
```

Comment:
Just a heads up....we have a pull request out to move propsTable and markdown into their own packages. We might need to update depending on who merges first.
https://github.com/cerner/terra-ui/pull/100